src/components/User.spec.js

```diff
@@ -0,0 +1,14 @@
+import { render, screen } from '@testing-library/react'
+import User from './User'
+
+const data = require('../data/data.json')
+
+const users = data
+
+describe('User', () => {
+  it('renders a list with personal data and skills', () => {
+    render(<User userData={users.map(user => user)} />)
```

Comment:
This doesn't work. Currently your `User` doesn't receive any props. It works because you imported the data in your `User` component directly. Components should be small, independent pieces of your app: https://reactjs.org/docs/components-and-props.html